Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -794,3 +819,36 @@ open class ComposerVC: _ViewController,
         }
     }
 }
+
+/// searchUsers does an autocomplete search on a list of ChatUser and returns users with `id` or `name` containing the search string
+/// results are returned sorted by their edit distance from the searched string
+/// distance is calculated using the levenshtein algorithm
+/// both search and name strings are normalized (lowercased and by replacing diacritics)
+func searchUsers(_ users: [ChatUser], by searchInput: String, excludingId: String? = nil) -> [ChatUser] {
```

Comment:
making this an open func means a breaking change (I am 100% sure we will change this in a few weeks to better support this feature)

Reply:
Ah I see, good point 👍